apps/heft/src/stages/StageBase.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import { Terminal } from '@rushstack/node-core-library';
+import { ITerminal } from '@rushstack/terminal';
 import { AsyncSeriesBailHook, SyncHook, AsyncSeriesHook } from 'tapable';
 import { HeftConfiguration } from '../configuration/HeftConfiguration';
```

Comment:
We might as well add our stylistic newline here between external and internal imports.